python/ray/tests/test_usage_stats.py

```diff
@@ -1165,6 +1165,13 @@ def ready(self):
         assert payload["python_version"] == python_version
         assert payload["schema_version"] == "0.1"
         assert payload["os"] == sys.platform
+        if sys.platform != "linux":
+            assert "glibc" not in payload
+        else:
+            import platform
+
+            assert payload["glibc"] == platform.libc_ver()[1]
```

Comment:
I think you also need to update the `schema` in the beginning of test_usage_stats.py